### ISSUE TITLE
Hypoplasia of the ureter 

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4666]
**Preferred term label:**
Hypoplasia of the ureter 

**Synonyms**
Ureter hypoplasia

**Definition (free text, please give PubMed ID)**
Underdevelopment of the ureter.

**Parent term (use hpo.jax.org/app)**
HP_0000069 'Abnormality of the ureter'

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.omim.org/entry/614080?search=614080&highlight=614080

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-5208-3432
